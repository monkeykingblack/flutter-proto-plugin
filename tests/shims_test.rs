use proto_pdk_test_utils::*;

mod flutter_tool {
    use super::*;

    #[cfg(not(windows))]
    generate_shims_test!("flutter-test", ["flutter", "dart"]);
}
