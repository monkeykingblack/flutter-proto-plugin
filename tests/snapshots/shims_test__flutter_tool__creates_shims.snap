---
source: tests/shims_test.rs
assertion_line: 7
expression: "std :: fs ::\nread_to_string(sandbox.path().join(\".proto/shims/registry.json\")).unwrap()"
---
{
  "dart": {
    "alt_bin": true,
    "parent": "flutter-test"
  },
  "flutter": {}
}
